### ISSUE TITLE
double const in iopll_freq_config

### ISSUE THREAD

**Issue by zhouyuan** [OPAE/opae-sdk#826]
```
diff --git a/libopae/src/usrclk/user_clk_iopll_freq.h b/libopae/src/usrclk/user_clk_iopll_freq.h
index e64ee2b..8b638d3 100755
--- a/libopae/src/usrclk/user_clk_iopll_freq.h
+++ b/libopae/src/usrclk/user_clk_iopll_freq.h
@@ -58,7 +58,7 @@ struct iopll_config {
 };

 // Reference frequency: 100MHz
-const struct iopll_config const iopll_freq_config[] = {
+const struct iopll_config iopll_freq_config[] = {
        { 0 }, // Freq 0 not configured
        { 0 }, // Freq 1 not configured
        { 0 }, // Freq 2 not configured
```

**Comment by tswhison** [comment]
Thank you for pointing this out.  We have also encountered this, and it should be fixed in the next release.

**Comment by tswhison** [comment]
Fixed in OPAE 1.2.
